src/main/lighting/AmbientLight.java

```diff
@@ -0,0 +1,25 @@
+package lighting;
+
+import primitives.Colour;
+
+/**
+ * This light models the small ammounts of light which reflected so many times that their source is indeterminable. All
+ * geometry intersections will be affected by at least this light.
+ *
+ * @author Abraham Murciano
+ * @author Eli Levin
+ */
+public class AmbientLight {
+
+	/** The colour of the ambient light. */
```

Comment:
color*

Reply:
https://tenor.com/view/english-do-you-speak-it-motherfucker-jules-pulp-gif-17700680

Reply:
Also, see #25